build.gradle

```diff
@@ -241,7 +241,10 @@ subprojects {
     }
   }
 
-  def shouldUseJUnit5 = ["clients", "examples", "log4j-appender", "raft", "test-utils", "tools"].contains(it.project.name)
+  // Remove the relevant project name once it's converted to JUnit 5
+  def shouldUseJUnit5 = !(["api", "basic-auth-extension", "connect", "core", "file", "generator",
+    "json", "mirror", "mirorr-client", "runtime", "transform", "examples", "streams-scala",
```

Comment:
```examples``` is already in junit 5 (bed4c6a33b16b96c1a20f47d818f7a44eea33fa9)

Reply:
oh, there is another module called ```examples```

Reply:
It doesn't have tests though, so I can probably remove it. Let me check.

Reply:
Fixed.